airbyte-server/src/main/java/io/airbyte/server/apis/DestinationOauthApiController.java

```diff
@@ -10,26 +10,33 @@
 import io.airbyte.api.model.generated.OAuthConsentRead;
 import io.airbyte.api.model.generated.SetInstancewideDestinationOauthParamsRequestBody;
 import io.airbyte.server.handlers.OAuthHandler;
+import io.micronaut.context.annotation.Context;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Post;
 import java.util.Map;
 import javax.ws.rs.Path;
 import lombok.AllArgsConstructor;
 
-@Path("/v1/destination_oauths")
+@Controller("/api/v1/destination_oauths")
 @AllArgsConstructor
```

Comment:
Nit:  we should remove this in favor of an explicit constructor that takes in the `OAuthHandler` as a parameter.  It's not a big deal, but the Lombok generated constructor doesn't allow us to annotate the parameters to the constructor, should we need them for Micronaut dependency injection.

Reply:
Done, do you know how Micronaut works with records?

Reply:
I don't, though I would doubt that we would want to make a record a singleton.  Records are generally meant to encapsulate data and do not typically provide any sort of business logic.